src/styles/StyleUtils.ts

```diff
@@ -471,12 +398,8 @@ function getBadgeColorStyle(success, error, isPressed = false, isAdHoc = false)
 
 /**
  * Generate a style for the background color of the button, based on its current state.
- *
- * @param {String} [buttonState] - One of {'default', 'hovered', 'pressed'}
- * @param {Boolean} isMenuItem - whether this button is apart of a list
```

Comment:
You should leave the JSDoc if the parameter has some description, just remove the content with `{...}`

Reply:
Done